code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -805,3 +806,32 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 	amount = 20
 /obj/item/stack/sheet/sandblock/five
 	amount = 5
+
+/*
+ * Titanium Alloy
+ */
+/obj/item/stack/sheet/tialloy
+	name = "titanium alloy"
+	singular_name = "titanium alloy sheet"
+	desc = "This sheet is an alloy of plasteel and titanium."
+	icon_state = "sheet-tialloy"
+	item_state = "sheet-tialloy"
+	custom_materials = list(/datum/material/titanium=MINERAL_MATERIAL_AMOUNT, /datum/material/iron=MINERAL_MATERIAL_AMOUNT, /datum/material/plasma=MINERAL_MATERIAL_AMOUNT)
```

Comment:
format this like 
```
= list(
	thing1,
	thing2,
)
```

Reply:
no other composite material does it like this